java/src/main/resources/template.properties

```diff
@@ -19,7 +19,9 @@ hive.gcs.output.format=avro
 hive.partition.col=
 ## Optional: Write mode to gcs append/overwrite/errorifexists/ignore
 hive.gcs.save.mode=overwrite
-
+## Optional: Below mentioned two properties needs to be used whenever there is a need to perform sql based transformations before loading data
+hive.gcs.temp.table=<temporary_view_name>
```

Comment:
property value must be blank otherwise it will not be null in code.
Only default values of properties must be defined in code.

Reply:
Resolved